rectools/visuals/visual_app.py

```diff
@@ -470,37 +713,26 @@ class ItemToItemVisualApp(VisualAppBase):
 
     def __init__(
         self,
-        recos: TablesDict,
+        reco: tp.Union[pd.DataFrame, TablesDict],
         item_data: pd.DataFrame,
-        selected_items: tp.Dict[tp.Hashable, tp.Hashable],
+        selected_items: tp.Dict[tp.Hashable, ExternalId],
         n_random_items: int = 0,
         auto_display: bool = True,
         formatters: tp.Optional[tp.Dict[str, tp.Callable]] = None,
         rows_limit: int = 20,
         min_width: int = 100,
     ) -> None:
-        super().__init__(
-            recos=recos,
+        data_storage = AppDataStorage.from_raw(
```

Comment:
Not critical, but it's recommended  to put `super()....` call in the 1st place

Reply:
Fixed